includes/meta.php

```diff
@@ -41,9 +41,23 @@ function sticky_post_for_term( $post_id, $term_id ) {
  * Register the Sticky meta box on applicable post types.
  */
 function register_meta_boxes() {
-	$taxonomies = array_keys( get_taxonomies( [
+
+	/**
+	 * Retrieve an array of post types that should be eligible for taxonomy-based stickiness.
+	 *
+	 * @param array $post_types Post types that should be able to use Sticky Tax.
+	 */
+	$post_types = apply_filters( 'stickytax_post_types', array( 'post' ) );
+
+	// Bail if we have no post types.
```

Comment:
Good call. Might even make sense to drop this filter and instead register post type support for 'sticky-tax', then query post types using [`get_post_types_by_support()`](https://developer.wordpress.org/reference/functions/get_post_types_by_support/), but a filter works for now.